src/renderer/pages/sshsetup/CloneRepo.js

```diff
@@ -62,11 +67,10 @@ const CloneRepo = observer(() => {
     }
   }, [desktopFolder]);
 
-  // Close all notifications on unmount.
-  // Using unmount method of useEffect() to remove all notifications displayed.
   React.useEffect(() => {
     return () => {
       toaster.closeAll();
+      lottie.destroy();
```

Comment:
Couldn't figure out another way to avoid importing `lottie` in this screen. We could export function like `stopAnimation()` apart from animation instance?

Reply:
Done 💯